WeakAuras/Types.lua

```diff
@@ -1321,7 +1321,10 @@ WeakAuras.anim_translate_types = {
   shake = L["Shake"],
   bounce = L["Bounce"],
   bounceDecay = L["Bounce with Decay"],
-  custom = L["Custom Function"]
+  custom = L["Custom Function"],
+  easeIn = L["Ease In"],
+  easeOut = L["Ease Out"],
+  easeInOut = L["Ease In and Out"]
```

Comment:
Keep custom at the end of the list for only code aesthetics reasons.

Reply:
That gif is mine, just demonstrating the amount of easing I'm adding. 
The one without easing from Discord that inspired this is <https://cdn.discordapp.com/attachments/218087027042811905/694561955016343573/Rogue_WA_Slide.gif> from user <Holte#6934>